addons/mail/static/src/core/common/record.js

```diff
@@ -419,19 +427,20 @@ export class Record {
     }
     /**
      * @template {keyof import("model ").Models} M
-     * @param {M} modelName
+     * @param {M} targetModel
      * @returns {import("models").Models[M]}
      */
-    static one(modelName) {
-        return [ONE_SYM, modelName];
+    static one(targetModel) {
+        return [ONE_SYM, { targetModel }];
     }
     /**
      * @template {keyof import("model").Models} M
-     * @param {M} modelName
+     * @param {M} targetModel
+     * @param {(r) => void} [onDelete]
```

Comment:
This should be documented. I don't understand what delete means, is it the deletion of current record? Is it deletion of related record? Is it deletion from the relation?

I'm still not convinced that delete is a better word than unlink for relations.